api/openapi.generated.yml

```diff
@@ -27,25 +27,28 @@ paths:
                     $ref: '#/components/schemas/Healthcheck'
                   status_code:
                     type: integer
+                    minimum: -2147483648
+                    maximum: 2147483647
```

Comment:
I might want to remove what added this - this was porting over something that by default limits integers to the values the Postgres DB supports, but it might be more confusing to have this on fields like status code